auth/api/iam/openid4vp.go

```diff
@@ -162,12 +171,16 @@ func (r *Wrapper) handlePresentationRequest(params map[string]string, session *S
 // handleAuthConsent handles the authorization consent form submission.
 func (r *Wrapper) handlePresentationRequestAccept(c echo.Context) error {
 	// TODO: Needs authentication?
-	var session *Session
-	if sessionID := c.FormValue("sessionID"); sessionID != "" {
-		session = r.sessions.Get(sessionID)
+	sessionID := c.FormValue("sessionID")
+	if sessionID == "" {
+		return errors.New("missing sessionID parameter")
 	}
-	if session == nil {
-		return errors.New("invalid session")
+
+	var session Session
+	sessionStore := r.storageEngine.GetSessionDatabase().GetStore(sessionExpiry, session.OwnDID.String(), "session")
```

Comment:
`session` is too ambiguous. `openid4vp`,`session` would be better when you debug and list keys in session storage.

Reply:
For the authorized code flow/OpenID4VP, this is more in POC state than actually production-grade, so I expect it to change in future. But I'll change the keys. Idea for keys:

- `openid.(did).session` (can be OpenID4VP/SIOPv2 or OpenID4VCI)
- `s2s.(did).code`